### ISSUE TITLE
linux-arm64  grabber.close()  the memory do not release 

### ISSUE THREAD

**Issue by wangjia5693** [bytedeco/javacv#1847]
as the title describe ,i use Pointer.formatBytes(Pointer.physicalBytes()) to mark the memory usage , when i call the  grabber.close() and record.close(), in my macos,it release the memory ,but in linux-arm64 ,it can't

**Comment by saudet** [comment]
On Linux, we may need to call Pointer.trimMemory() to release the memory to the system.

**Comment by wangjia5693** [comment]
> On Linux, we may need to call Pointer.trimMemory() to release the memory to the system.

thank you for you help ,i will have a  try ; but i am a beginner of this，can you give me a  demo or more reference of this, thanks a lot

**Comment by saudet** [comment]
If you're looking for the API documentation, it's available here:
http://bytedeco.org/javacpp/apidocs/org/bytedeco/javacpp/Pointer.html#trimMemory--

**Comment by wangjia5693** [comment]
> If you're looking for the API documentation, it's available here: http://bytedeco.org/javacpp/apidocs/org/bytedeco/javacpp/Pointer.html#trimMemory--

i am not very clear,Pointer.trimMemory() is a static private method ,how can i call it

**Comment by saudet** [comment]
Right, it's a private method. It's not something users usually need to worry about. It currently only does something on Linux, and just essentially calls malloc_trim(). In any case, please try to make it public and see if that does what you want it to do. If it works, please open a pull request with the requested changes: https://github.com/bytedeco/javacpp/pulls

**Comment by saudet** [comment]
It's also possible that your version of the JDK calls malloc_trim() on System.gc(), so you could also try to call that and see if it does what you need.

**Comment by Vansonccc** [comment]
I met the same problem, how did you solved this problem?
